.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "src/external/rawspeed"]
 	path = src/external/rawspeed
-	url = https://github.com/darktable-org/rawspeed.git
-	branch = develop
+	url = https://github.com/sarunasb/rawspeed.git
+	branch = om
```

Comment:
The submodule change shouldn't have been committed obviously...

Reply:
Unfortunately it became obvious enough for me only after the fact. Should now be corrected.